eu.codepoems.xl-converter.yaml

```diff
@@ -0,0 +1,293 @@
+                commit: ed738e842d2fbdf2d6459e39267a633c4a9b2f5d
+                x-checker-data:
+                  type: git
+                  tag-pattern: ^v([\d.]+)$
+          - name: perl
+            buildsystem: simple
+            build-commands:
+              - ./Configure -des -Duseshrplib -Dman3dir=none -Dman1dir=none -Dprefix=/app
+              - make
+              - make install
+            cleanup:
+              - /bin/corelist
+              - /bin/cpan
+              - /bin/enc2xs
+              - /bin/encguess
+              - /bin/h2ph
+              - /bin/h2xs
+              - /bin/instmodsh
+              - /bin/json_pp
+              - /bin/libnetcfg
+              - /bin/perl5*
+              - /bin/perlbug
+              - /bin/perldoc
+              - /bin/perlivp
+              - /bin/perlthanks
+              - /bin/piconv
+              - /bin/pl2pm
+              - /bin/pod2html
+              - /bin/pod2man
+              - /bin/pod2text
+              - /bin/pod2usage
+              - /bin/podchecker
+              - /bin/prove
+              - /bin/ptar
+              - /bin/ptardiff
+              - /bin/ptargrep
+              - /bin/shasum
+              - /bin/splain
+              - /bin/streamzip
+              - /bin/xsubpp
+              - /bin/zipdetails
+              - '*.pod'
+            build-options:
+              cflags: -fPIC
+              ldflags: -fpic
+            post-install:
+              - find ${FLATPAK_DEST}/lib/perl5/ -name \*.so -exec chmod u+w {} \;
+            sources:
+              - type: archive
+                url: https://www.cpan.org/src/5.0/perl-5.40.1.tar.xz
+                sha256: dfa20c2eef2b4af133525610bbb65dd13777ecf998c9c5b1ccf0d308e732ee3f
+                x-checker-data:
+                  type: anitya
+                  project-id: 13599
+                  stable-only: true
+                  url-template: https://www.cpan.org/src/${major}.0/perl-$version.tar.xz
+              - type: script
+                dest-filename: configure
+                commands:
+                  - exec ./configure.gnu $@
+
+  - name: xl-converter
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 xl-converter/misc/eu.codepoems.xl-converter.metainfo.xml /app/share/metainfo/eu.codepoems.xl-converter.metainfo.xml
+      - desktop-file-edit --set-icon=eu.codepoems.xl-converter xl-converter/misc/xl-converter.desktop
+      - install -Dm644 xl-converter/misc/xl-converter.desktop /app/share/applications/eu.codepoems.xl-converter.desktop
+      - install -Dm644 xl-converter/assets/icons/logo.svg /app/share/icons/hicolor/scalable/apps/eu.codepoems.xl-converter.svg
+      - install -Dm755 run.sh /app/bin/run.sh
+      - mkdir -p /app/xl-converter
+      - cp -r xl-converter/* /app/xl-converter/
+      - mkdir -p /app/xl-converter/bin/linux
+      - |
+        echo '#!/bin/bash
+        exec /usr/bin/jpegtran "$@"' > /app/xl-converter/bin/linux/jpegtran
```

Comment:
use a script source for this https://docs.flatpak.org/en/latest/module-sources.html#shell-and-script-sources